src/main/scala/IOService/Http4sTestServer.scala

```diff
@@ -0,0 +1,109 @@
+package IOService
+import cats._
+import cats.effect._
+import cats.implicits._
+import org.http4s.circe._
+import org.http4s._
+import io.circe.generic.auto._
+import io.circe.syntax._
+import org.http4s.dsl._
+import org.http4s.implicits._
+import org.http4s.server.blaze.BlazeServerBuilder
+import cats.Monad
+import cats.effect.unsafe.implicits.global
+import cats.effect.{Concurrent, ExitCode, IO, IOApp}
+import org.http4s.server.Router
+import org.scalacheck.Gen
+import poker.Deck.startingDeck
+import poker.{Card, Street}
+import poker.Street._
+
+import scala.collection.mutable
+
+object Http4sTestServer extends IOApp {
+
+  /*
+  - Get all the players and cards of current Street - the board state
+  - Post - deal the hole cards -  preflop state
+   */
+
+  // Request -> F[Option[Response]]
+  // HttpRoutes[F] ^^^
+
+  // Get /flop
+  // GET /turn
+  // GET /river
+  // GET /players
+
+  // ??? Need a DB/state
+
+  def genPreFlopBoard(numPlayers: Int): Gen[Preflop] =
```

Comment:
Curious why you are using ScalaCheck outside of `test`

Reply:
Per the docs, you're free to use the generators whenever, not just for test.  (But this is just temporary, because I wanted to see a response in my browser as this is my first time using Http4s)  It was an easy way to generator some data.  But it is definitely not intended to remain here. I will create a proper test which will probably do something similar.